src/PayID/pay-id-client.ts

```diff
@@ -74,3 +73,5 @@ export default class PayIDClient {
     return address
   }
 }
+
+export { PayIDErrorType, PayIDError }
```

Comment:
These are exported from `pay-id-error.ts`. Why export them from here? is this a common pattern? 

in index.ts you pull them directly from the error file?

Reply:
this was a mistake; fixed